apps/frontend/core-dapp/src/features/ppo/stake/UnstakeStore.ts

```diff
@@ -1,18 +1,21 @@
 import { CheckboxChangeEvent } from 'antd/lib/checkbox'
+import { BigNumber } from 'ethers'
 import { makeAutoObservable } from 'mobx'
 import { TRANSACTION_SETTING } from '../../../lib/constants'
 import { RootStore } from '../../../stores/RootStore'
 import { validateNumber } from '../../../utils/number-utils'
 
-const FEE_MOCK = 7.5
-
 export class UnstakeStore {
   confirm = false
   currentUnstakingValue = TRANSACTION_SETTING.DEFAULT_AMOUNT
-  fee = FEE_MOCK
 
   get isCurrentUnstakingValueValid(): boolean {
-    return Boolean(this.currentUnstakingValue) // TODO: add staked value check
+    const stakedPPO = this.root.ppoStakingStore.balanceData?.raw
+    if (!stakedPPO) return false
+    return (
+      BigNumber.from(stakedPPO).toNumber() >= this.currentUnstakingValue &&
```

Comment:
A better approach would be to have values like balanceData.raw converted to BigNumber inside getMockedBalance, so we don't have to do BigNumber.from everywhere, which isn't something we should do when actual SC is integrated.

Also it's kinda redundant to write BigNumber.from(anotherBigNumber)

```suggestion
      stakedPPO.toNumber() >= this.currentUnstakingValue &&
```